lib/bunyan.js

```diff
@@ -995,10 +995,16 @@ Logger.stdSerializers.res = function res(res) {
 function getFullErrorStack(ex)
 {
     var ret = ex.stack || ex.toString();
-    if (ex.cause && typeof (ex.cause) === 'function') {
-        var cex = ex.cause();
+    if (ex.cause) {
+        var cex;
+        if (typeof (ex.cause) === 'function') {
+            cex = ex.cause();
+        }
+        if (typeof (ex.cause) === 'object') {
+            cex = ex.cause;
+        }
         if (cex) {
-            ret += '\nCaused by: ' + getFullErrorStack(cex);
+            ret += '\ncaused by ' + getFullErrorStack(cex);
         }
```

Comment:
Why this change? How does this "integrate better with the default stack trace formatter"? I'm not familiar with the "default" stack trace formatting you are referring to.

Also, if we decide to keep this "caused by" change, there is at least one test case that will need to be updated.